src/Microsoft.AspNetCore.Server.IntegrationTesting/breakingchanges.netcore.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+      "TypeId": "public abstract class Microsoft.AspNetCore.Server.Testing.ApplicationDeployer : Microsoft.AspNetCore.Server.Testing.IApplicationDeployer",
```

Comment:
The baseline should be for 1.0.0, when this project / assembly were named Microsoft.AspNetCore.Server.Testing. Renaming the assembly was _the_ breaking change. The baseline file should have been removed then.